### ISSUE TITLE
Feature Request :: show how many tokens in ROI / APR

### ISSUE THREAD

**Issue by shoegaze503** [PolyPup-Farm/polypup-ui-alpha-testers#3]
In the ROI and APR section it shows how much $ you should expect over the course of one day or one year: awesome!

However, I think some folks are also going to want to see how many tokens they would expect.  Maybe could fit between % and $ amount.

<img width="1162" alt="Screen Shot 2021-08-29 at 10 25 50 PM" src="https://user-images.githubusercontent.com/89763889/131289699-3b3c96ba-7d87-47c2-a33e-2177ab5f08c4.png">

**Comment by Polypup** [comment]
Good idea, gotta figure out how to make it look good without it being too cluttered.

**Comment by fire-farmer** [comment]
I'll just add this here as I feel it would fall under the same category. Do you think it possible to show your current dollar cost average for whatever token you're viewing?
